dev/com.ibm.ws.request.timing.jdbc/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -28,5 +28,8 @@ slowRequestThreshold.desc=Duration of time that a JDBC request can run before be
 hungRequestThreshold.name=Hung request threshold
 hungRequestThreshold.desc=Duration of time that a JDBC request can run before being considered hung.
 
+enableThreadDumps=Enable thread dumps
+enableThreadDumps.desc=Indicates whether thread dumps will be created if a hung request is detected. A value of false causes thread dumps to not be created when a hung request is detected. A value of true, by default, will cause thread dumps to be created for hung requests.
```

Comment:
Indicates whether thread dumps are created when a hung request is detected. When this value is set to true (default), thread dumps are created. When set to false, thread dumps are not created.